pyccel/parser/semantic.py

```diff
@@ -724,7 +724,7 @@ def get_type_description(self, var, include_rank = True):
         """
         dtype = var.dtype
         prec  = get_final_precision(var)
-        descr = f'{dtype}(kind={prec})'
+        descr = f'{dtype}{(prec * 2 if dtype == NativeComplex() else prec) * 8 if prec else ""}'
```

Comment:
I think it's a good idea to use `isinstance` when trying to compare the identity of something

Reply:
you can just change the comparison to `isinstance(dtype, NativeComplex)`